assets/js/swedbank-pay-design-guide-theme.js

```diff
@@ -11,40 +12,41 @@
         flowchart: {
             useMaxWidth: false
         }
-    });
+    }
+    mermaid.initialize(configObject);
 
-    mermaid.init(undefined, 'code.language-mermaid');
+    mermaid.init(configObject, "code.language-mermaid");
 })();
 
 // Initialize sidebar navigation scroll activation
-(function() {
+(function () {
     var headings = document.querySelectorAll("h2");
     var tocLinks = document.querySelectorAll("nav.doc-toc ul li a");
 
-    var getPosition = function(el) {
-      if (el) {
-        var bodyRect = document.body.getBoundingClientRect();
-        var elemRect = el.getBoundingClientRect();
+    var getPosition = function (el) {
+        if (el) {
+            var bodyRect = document.body.getBoundingClientRect();
+            var elemRect = el.getBoundingClientRect();
 
-        return elemRect.top - bodyRect.top;
-      }
+            return elemRect.top - bodyRect.top;
+        }
 
-      return null;
+        return null;
     };
 
-    window.addEventListener("scroll", function() {
+    window.addEventListener("scroll", function () {
         var buffer = document.body.clientHeight * 0.1;
         var currentPos = window.pageYOffset + buffer;
 
         // TODO: Probably a stupid way to compute "how far left can we scroll until
         //       we reach the bottom of the page", but it seems to work.
         var scrollDistanceFromBottom = document.documentElement.scrollHeight
-                                     - document.documentElement.scrollTop
-                                     - document.body.clientHeight
-                                     - buffer;
+            - document.documentElement.scrollTop
+            - document.body.clientHeight
+            - buffer;
 
         if (scrollDistanceFromBottom <= 0) {
-            for (var link of tocLinks) {
+            for (let link of tocLinks) {

```

Comment:
Since this is client-side JavaScript that is not transpiled in any way, I would like to stay conservative and use `var`.

```suggestion
            for (var link of tocLinks) {
```